### ISSUE TITLE
I tried training on the raccoon dataset and it's not working right. What might be the issue ?

### ISSUE THREAD

**Issue by andohuman** [experiencor/basic-yolo-keras#204]
![raccoon-32_detected](https://user-images.githubusercontent.com/26820345/37835310-e080485e-2ed5-11e8-9018-0b09e30d325a.jpg)

**Comment by experiencor** [comment]
@andohuman Good progress. You are getting there. Just need a little more patience to play with the 4 scales. You will understand YOLO a lot better if you know how to tune these scales.

**Comment by andohuman** [comment]
@experiencor Thanks. Is there any tips you might give to understand your loss function better ? I've tried understanding it for a long time but I can't comprehend it. Also I assume by 4 scales you mean xy, wh, conf and class ?

**Comment by andohuman** [comment]
I've found the issue more or less. I was training only the last conv layer and not the entire net. Once i fixed that, i started getting better results.
